Tests/CreateAPITests/GenerateOptionsTests.swift

```diff
@@ -359,6 +359,24 @@ final class GenerateOptionsTests: GenerateBaseTests {
         // THEN
         try compare(package: "petstore-change-namespace-when-operations-style")
     }
+    
+    func testPetstoreInternalAccessControl() throws {
+        // GIVEN
+        let command = try Generate.parse([
+            pathForSpec(named: "petstore", ext: "yaml"),
+            "--output", temp.url.path,
+            "--package", "petstore-internal-access-control",
+            "--config", config("""
+            access: internal
+            """, ext: "yaml")
+            ])
+            
+        // WHEN
+        try command.run()
+        
+        // THEN
+        try compare(package: "petstore-internal-access-control")
```

Comment:
Can we please generate the equivalent of what was in `edgecases-change-access-control`? It makes things easier to review because the diff of the generated package won't have changed. 

Also, before recording the snapshots. Please can you delete the contents of the Expected folder (not the folder itself) so that we cleanup unrelated files. For example, looking at this branch the following shouldn't be there:

<img width="1509" alt="Screenshot 2022-08-06 at 13 00 02" src="https://user-images.githubusercontent.com/482871/183246119-7c6623e4-162a-4058-bdde-06b183c6022c.png">

Reply:
By "generate the equivalent" do you mean the following config:
```yaml
access: ""
```

Or do you mean the keep the name of the package the same? The new test is no longer an edge case and follows the other test naming conventions, so the big diff is expected but I just didn't expect that large of a deletion diff from the edge cases package.

Reply:
Changing the package name is ok since it's just a rename. 

But changing the schema is unnecessary for what we are doing in this PR so it is easier for us to focus on the actual change (adding the enum with zero breaking changes) if we don't generate a lot of extra noise in the diff. 

I know in this particular PR its all pretty low risk, but if I could glance and see from the diff that there are zero changes to the generated output then I can be easily satisfied that the change is good and we will have merged it already. Changing the schema means that there are now loads of deletions and a bunch of new files that I have to actually look at and think about to know for sure if this is doing what we expect. 

It's worth getting into this practice of separating the changes and minimising the diff so that we can get things through quicker with more confidence 🙏

Reply:
The structure of the snapshot tests need an overall rethink anyway (including practices on easily generating smaller outputs for testing minor details) so I wouldn't worry too much about trying to change things right now